### PR TITLE
Initial Uncontained.io repository updates

### PR DESCRIPTION
**Pull request by JaredBurck**
#### What is this PR About?
There are numerous updates in this PR for the repository.  Please review the following files that were either added or updated.  Consider these files a rough draft with sections that may need updates or to be reworked.  Comment or make suggestions anywhere you see a need for improvement or clarification:

README.md - updated to reflect repository information
CONTRIBUTING.md - moved from README.md and updated guidance and setup information
CODE_OF_CONDUCT.md - added code of conduct for uncontain.io community
LICENSE -  updated license from cc-by-4.0 to cc-by-sa-4.0
notices.md - add notices for legal disclaimer, licenses, and permissions
.github/PULL_REQUEST_TEMPLATE.md - added and updated PR template
.github/stale.yml - GitHub protobot plugin configuration file, currently commented out
docs/content_model.md - added new doc that describes the content model used for site
docs/new_guides.md - added new doc for creating a new guide
docs/personas.md - added new doc for describe community personas
docs/roadmap.md - added new doc to outline the direction of the uncontained.io project
docs/style_guide.md - moved and updated the style guide recommendations

#### How should we test or review this PR?
Review the new and updated pages above and provide feedback, comments, and/or improvements.

#### Is there a relevant Trello card or Github issue open for this?
https://trello.com/c/oibNh5aJ/85-update-uncontainedio-repository-to-provide-better-information-about-project-and-how-this-repository-works-and-is-maintained

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----

- [x] Have you followed the [contributing guidelines](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the uncontained.io guides?

-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by rdebeasi** [comment]
Looks awesome - I really like the new guides, code of conduct, and such.

One question about this line:
> First and foremost, this project is a forum to discuss DevOps technical practices and container automation approaches using Red Hat commercial open source products, then document them in a guide when we've found consensus. 

In the past few moths, we've discussed broadening the site's coverage to things like emerging technologies, DevOps tools from any vendor, etc. Did the CoP ultimately decide not to do that? I think this site would be more appealing for the open source community if it could cover topics such as Docker or Jenkins independently from Openshift. I know this is a fine line, because the site is also focused on getting operationalizing, and we probably wouldn't recommend running a production cluster on, say, OpenShift Origin or Fedora.

**Comment by etsauer** [comment]
Ryan/Jared,

I agree, and think that that is still the direction. I think we just need
to adjust the wording there.

**Comment by JaredBurck** [comment]
Agreed.  Thanks @rdebeasi for the feedback!  Also, I would like to discuss the content model of this site with you to make sure it aligns our content types and style guide.

I have also made some other assumptions, like using the word "guide" that we may want to review as well. I would also like the personas to line up with the ninja program in some way.

I also have updated the `What is this PR About?` section to be more clear on my ask.

File: docs/content_model.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This content was originally created and curated by Red Hat's PaaS Community of Practice, and covers topics that are very relevant to Red Hat users, but it is not specific to Red Hat products.
+
+For content that is specific to Red Hat products, see:
+
+- docs.openshift.com - gets existing users unstuck and back to work utilizing OpenShift Container Platform.
+- access.redhat.com -  support, documentation, and more about all Red Hat specific products and services for all users.
+
+Everything written in the guides should fall into one of the following categories.
+
+## Article Guides
```
Comment (etsauer) [inline docs/content_model.md]: New breakdown of Content Model:

* Guides - Long form documents that dive deep on a topic and have multiple contributors
* Articles - Short form documents, opinion pieces, ideas from individuals
  ** Tutorials - specific type of article with specific expected content structure.
  ** maybe other?

File: docs/content_model.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Link Page Guides
+
+A link page is simply a guide that provides a list of links to great resources around the web. Write down the title of the article, hyperlink it, and number it. It’s helpful to add your own blurb or introduction for each link you provide. Although not necessary, it’s a good way to put your own thought leadership on a topic or add a bit of value to the discussion.
+
+## FAQ
+
+An FAQ tackles a complex topic where a reader is likely coming in with specific questions (for example, "How do I...?" or "Troubleshooting OpenShift"). Whereas article and tutorial guides aim to teach the entire concept, an FAQ respects a reader's desire to jump around, get the information they need, and leave. The table of contents is especially important in an FAQ, because the page isn't meant to be read from beginning to end. FAQs might also be longer than a concept guide, because of the jump-around navigation.
+
+## Design Elements
```
Comment (etsauer) [inline docs/content_model.md]: I beleive everything below should be `###`

**Comment by etsauer** [comment]
[Uncontained Repo Documentation](https://trello.com/c/vulz3g5n/117-uncontained-repo-documentation)

**Comment by JaredBurck** [comment]
@etsauer resolved conflicts

**Pull request merged**